<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
SPDX-FileCopyrightText: none
SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-KDE-Accepted-GPL
 </author>
 <class>CustomCamcorderProxy_UI</class>
 <widget class="QDialog" name="CustomCamcorderProxy_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="13" colspan="4">
    <widget class="QListWidget" name="profiles_list">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="l_relPathOrigToProxy">
     <property name="text">
      <string>Relative path from clip to proxy:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="le_relPathOrigToProxy"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="l_prefix_clip">
     <property name="text">
      <string>Prefix of clip:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="le_prefix_clip"/>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="l_suffix_clip">
     <property name="text">
      <string>Suffix of clip:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="le_suffix_clip"/>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="l_relPathProxyToOrig">
     <property name="text">
      <string>Relative path from proxy to clip:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="le_relPathProxyToOrig"/>
   </item>
   <item row="8" column="4">
    <widget class="QLabel" name="l_prefix_proxy">
     <property name="text">
      <string>Prefix of proxy:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QLineEdit" name="le_prefix_proxy"/>
   </item>
   <item row="10" column="4">
    <widget class="QLabel" name="l_suffix_proxy">
     <property name="text">
      <string>Suffix of proxy:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QLineEdit" name="le_suffix_proxy"/>
   </item>
   <item row="13" column="0">
    <widget class="QToolButton" name="button_add">
     <property name="toolTip">
      <string>Add new profile</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QToolButton" name="button_remove">
     <property name="toolTip">
      <string>Delete profile</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="edit-delete">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="3">
    <widget class="QToolButton" name="button_reset">
     <property name="toolTip">
      <string>Reset all profiles</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="edit-reset">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="5">
    <widget class="KMessageWidget" name="proxy_message">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="closeButtonVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomCamcorderProxy_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomCamcorderProxy_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
